helix-term/src/keymap.rs

```diff
@@ -502,6 +502,11 @@ impl Default for Keymaps {
             },
 
             "\"" => select_register,
+            "|" => shell_pipe,
+            "A-|" => shell_pipe_ignore,
+            "!" => shell_insert,
+            "A-!" => shell_append,
+            "$" => shell_filter,
```

Comment:
I think a better option would be to `s` and then some keys to switch it to shell command, what do you think?

Reply:
What? I'm not sure I understand what you're saying.

Reply:
Since we have `s` which opens a prompt for select. Is it better to have a key in the prompt where we can change that to use command filter instead (we could also have infobox there)? So we don't have to have a specific key just for this and can make use of existing prompt.

Reply:
I think `$` is okay for now, since it's a single keypress. I would rename the command to `keep_pipe` though to match the original